backend/src/services/model.ts

```diff
@@ -144,12 +148,18 @@ export async function getModelCard(
   version: number | GetModelCardVersionOptionsKeys,
 ) {
   if (version === GetModelCardVersionOptions.Latest) {
-    const card = (await getModelById(user, modelId)).card
+    const model = await getModelById(user, modelId)
```

Comment:
The changes in this function are no longer needed as we're not using authorisation anymore